configs/rover_configs.yaml

```diff
@@ -64,6 +64,37 @@ process_discovery:
         # The Process Labels, used to aggregate similar process from service entity
         # Multiple labels split by ","
         labels: ${ROVER_PROCESS_DISCOVERY_REGEX_SCANNER_LABELS:}
+  kubernetes:
+    # Is active the kubernetes process detector
+    active: ${ROVER_PROCESS_DISCOVERY_KUBERNETES_ACTIVE:false}
+    # Current node name in kubernetes environment
+    node_name: ${ROVER_PROCESS_DISCOVERY_KUBERNETES_NODE_NAME:}
+    # include namespaces, multiple namespace split by ",", if empty means including all namespaces
+    namespaces: ${ROVER_PROCESS_DISCOVERY_KUBERNETES_NAMESPACES:}
```

Comment:
Are node name and namespace able to detect automatically? Just confirming.

Reply:
The namespace could detect automatically. The node name needs to use environment inject, but It could be injected from the field. Follow: https://kubernetes.io/docs/tasks/inject-data-application/environment-variable-expose-pod-information/

Reply:
If it could be injected, I think we should do this in helm/operator

Reply:
Yes, I have added a use case, It includes this config. https://github.com/apache/skywalking-rover/pull/16/files#diff-4e991562fb17ac1ab40e3864607cd0a3e97a25ca328d00593219441e65e9f5dbR78-R81